schema/ContainerImage.schema.yaml

```diff
@@ -0,0 +1,102 @@
+title: ContainerImage
+'@id': stencila:ContainerImage
+extends: Thing
+role: primary
+status: experimental
+category: code
+description: A linux container image (such as provided by Docker or Singularity) that is used in scientific computing.
+properties:
+  annotations:
+    '@id': stencila:annotations
+    description: | 
+      One or more annotations (labels) for the container image, typically key value pairs represented as strings
```

Comment:
I can't help but wonder if there is an existing schema.org property related to this. We recently had a discussion wrt to `CreativeWork` `about` `keyword` and `genre` properties that may be of relevance or at least provide some inspiration #210.

Reply:
What about https://schema.org/PropertyValueSpecification?

Reply:
I think https://schema.org/PropertyValue should suffice - we already have that implemented here. For example of use (which would be similar here)

https://github.com/stencila/schema/blob/83fe1ba6c5c58bc228311574b8bdcca81de4c9a5/schema/Thing.schema.yaml#L23-L37